src/app/Instance/BridgeOverview/BridgeOverview.test.tsx

```diff
@@ -115,6 +133,51 @@ describe("Bridge Overview", () => {
     expect(onCreateProcessor).toHaveBeenCalledTimes(1);
   });
 
+  it("should check for delete functionality", async () => {
+    const onDeleteProcessor = jest.fn();
+
+    const { comp } = setupBridgeOverview({
+      processorList: processor,
+      bridgeStatus: ManagedResourceStatus.Ready,
+      onDeleteProcessor,
+    });
+
+    await waitForI18n(comp);
+
+    expect(comp.queryByText("Delete")).not.toBeInTheDocument();
+    expect(comp.getByRole("button", { name: "Actions" })).toBeInTheDocument();
+    fireEvent.click(comp.getByRole("button", { name: "Actions" }));
+    expect(comp.queryByText("Delete")).toBeInTheDocument();
+    fireEvent.click(comp.getByRole("menuitem", { name: "Delete" }));
+    expect(onDeleteProcessor).toHaveBeenCalledWith(
+      "a72fb8e7-162b-4ae8-9672-f9f5b86fb3d7",
+      "Processor one"
+    );
+    expect(onDeleteProcessor).toHaveBeenCalledTimes(1);
+  });
+
+  it("should check for edit functionality", async () => {
+    const onEditProcessor = jest.fn();
+
+    const { comp } = setupBridgeOverview({
+      processorList: processor,
+      bridgeStatus: ManagedResourceStatus.Ready,
+      onEditProcessor,
+    });
+
+    await waitForI18n(comp);
+
+    expect(comp.queryByText("Edit")).not.toBeInTheDocument();
+    expect(comp.getByRole("button", { name: "Actions" })).toBeInTheDocument();
+    fireEvent.click(comp.getByRole("button", { name: "Actions" }));
+    expect(comp.queryByText("Edit")).toBeInTheDocument();
+    fireEvent.click(comp.getByRole("menuitem", { name: "Edit" }));
+    expect(onEditProcessor).toHaveBeenCalledTimes(1);
+    expect(onEditProcessor).toHaveBeenCalledWith(
+      "a72fb8e7-162b-4ae8-9672-f9f5b86fb3d7"
+    );
+  });
+
```

Comment:
I can see the `should have ingress endpoint` test in this file. Can you add here the check that the skeleton is displayed when the ingress endpoint is missing?